### ISSUE TITLE
DNA quant for

### ISSUE THREAD

**Issue by yd-x** [benjjneb/decontam#158]
Hi Benjamin,

Thanks for your time on developing and maintaining this package. It is exactly what I need!

I have a metagenomic dataset. My quick question is: for the input DNA quantitation data, can I use the total read count in each sample?

Thanks,
Yuandong

**Comment by benjjneb** [comment]
> for the input DNA quantitation data, can I use the total read count in each sample?

No. In (almost all) cases, a step happens during library pooling where higher amounts of library product from low-DNA samples are included on the sequencer, with the goal of getting a similar number of reads from all samples. Thus read depth does not correspond to input DNA concentration.
